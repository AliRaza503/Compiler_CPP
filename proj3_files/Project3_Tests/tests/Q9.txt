{
    switch (+ * - x y z w) {
        case 1: {
            y = * x 1.5;
        }
    }
} 
