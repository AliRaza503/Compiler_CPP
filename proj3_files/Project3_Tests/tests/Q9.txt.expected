TYPE MISMATCH 4 C2
